regparser/builder.py

```diff
@@ -237,7 +238,7 @@ def tree_and_builder(filename, title, checkpoint_path=None, doc_number=None):
     else:
         raise ValueError("Building from text input is no longer supported")
 
-    for preprocessor in all_preprocs.ALL:
+    for preprocessor in class_paths_to_classes(settings.PREPROCESSORS):
```

Comment:
Sounds reasonable--the code isn't hard to understand as it is. I'll file an issue for a rainy day.